governance/ApiDeprecation.md

```diff
@@ -0,0 +1,121 @@
+# API Deprecation
+
+## Notice
+
+> Wondering if this applies to you? This process only applies to __public__ APIs. If you're sure consumers of your API will not be impacted, making an immediate change is okay.
```

Comment:
Shouldn't this also include data formats? Public API may not change, but we under the hood flip formats that cause incompatibility (ie sync this week, version was binary, then later it flips to JSON and I sync to latest, now my content is broken).